include/drivers/i2c.h

```diff
@@ -10,9 +10,12 @@
 #include <intrusive.h>
 #include <stdint.h>
 
+#define I2C_NUM_PINS 2
```

Comment:
I'd prefer if the `OPS` macro was always at the top of the file, as it's sort of a different category from the other macros.

Reply:
Fixed.